tests/inventory/pipelines/test_data/fake_gke_services.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Fake GKE services data."""
+
+
+FAKE_SERVER_CONFIG = {  
+       'validImageTypes':[  
+          'UBUNTU',
+          'COS'
+       ],
+       'defaultImageType': 'COS',
+       'validNodeVersions':[  
+          '1.8.6-gke.0',
+          '1.8.5-gke.0',
+          '1.8.4-gke.1',
+          '1.8.3-gke.0',
+          '1.8.2-gke.0',
+          '1.8.1-gke.1',
+          '1.7.12-gke.0',
+          '1.7.11-gke.1',
+          '1.7.11',
+          '1.7.10-gke.0',
+          '1.7.8-gke.0',
+          '1.6.13-gke.1',
+          '1.6.11-gke.0',
+          '1.5.7'
+       ],
+       'validMasterVersions':[  
+          '1.8.6-gke.0',
+          '1.8.5-gke.0',
+          '1.7.12-gke.0',
+          '1.7.11-gke.1'
+       ],
+       'defaultClusterVersion': '1.7.11-gke.1'
+    }
+
+
+# This is different from the api test data in the the server_config is included
+# in the cluster data.
+FAKE_GKE_SERVICES_MAP = {
+    'project1': [{
+    'addonsConfig':{  
+       'networkPolicyConfig':{  
+          'disabled': True
+       }
+    },
+    'clusterIpv4Cidr': '10.60.0.0/14',
+    'createTime': '2016-11-12T17:57:49+00:00',
+    'currentMasterVersion': '1.6.13-gke.1',
+    'currentNodeCount':2,
+    'currentNodeVersion': '1.4.5',
+    'endpoint': '104.197.231.35',
+    'initialClusterVersion': '1.4.5',
+    'instanceGroupUrls':[  
+       'https://www.googleapis.com/compute/v1/projects/cicd-prod/zones/us-central1-b/instanceGroupManagers/gke-redditmobile-canary--default-pool-1388e059-grp'
```

Comment:
The project name, cluster name and other details should be fake / anonymous.

Reply:
Done.  Good spot.